pkg/flow/flow.go

```diff
@@ -277,10 +277,12 @@ func (f *Flow) Run(ctx context.Context) {
 //
 // The controller will only start running components after Load is called once
 // without any configuration errors.
-func (f *Flow) LoadSource(source *Source, args map[string]any) error {
+func (f *Flow) LoadSource(src any, args map[string]any) error {
```

Comment:
was the type changed here to prevent some dependency? It's a little weird to my eyes to have an `any` param that is casted to another single type before use.

Reply:
Yes, this is a way I tried to break a cyclic dependency.

I can write up some more details tomorrow so we can brainstorm if there's a better way to deal with this.